### PR TITLE
docs: update component descriptions

### PR DESCRIPTION
**Pull request by christophenne**
## 📑 Description
Updating docs: Welcome page cleanup, components. 

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Review by thekubernaut — APPROVED** [review]

File: website/docs/03_components/01_client.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+Both kinds of user interface are supposed to provide the same functionality, and it's up to the user to choose their favorite way of doing things. 
 
+The client's tasks revolve around handling user input and consequently managing the required operations with the cluster (via the Kubernetes API) and the package repository. 
+For example, when installing a package, the client creates a `Package` custom resource via the Kubernetes API. When listing the available packages, it fetches the package list from the repository.
+
+## CLI
+
+The CLI commands make use of the [Cobra Library](https://github.com/spf13/cobra), which helps in crafting easy to use command line applications.
+
+## UI
```
Comment (pmig) [inline website/docs/03_components/01_client.md:13]: We should name our graphical UI **GUI** instead of just UI, as an CLI is technically also a UI

**Review by kosmoz — APPROVED** [review]

File: website/docs/03_components/02_package-repository.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # Package Repository
```
Comment (pmig) [inline website/docs/03_components/02_package-repository.md:1]: filename should be 03_package-repository

**Review by pmig — CHANGES_REQUESTED** [review]

File: website/docs/03_components/01_client.md (hunk 1 above)
Comment (christophenne) [reply website/docs/03_components/01_client.md:13]: fixed in https://github.com/glasskube/glasskube/pull/161

File: website/docs/03_components/02_package-repository.md (hunk 2 above)
Comment (christophenne) [reply website/docs/03_components/02_package-repository.md:1]: fixed in https://github.com/glasskube/glasskube/pull/161

**Pull request merged**